src/schema/v2/artists/mergeArtistsMutation.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ResolverContext } from "types/graphql"
+import { ArtistType } from "../artist"
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "MergeArtistsSuccess",
+  isTypeOf: (data) => data.id,
```

Comment:
**Question**: I wanted to ask regarding this usage as I am working on a similar PR and keep having issues with types in my tests. I see that in a number of _SuccessTypes_ we return the `.id` for `isTypeOf`. Do you happen to know why do we use `.id` here instead of `_type` as we do in the _FailureType_?

Reply:
Good question. I sort of breezed over this as I tried to learn from existing mutations in MP.

But my understanding is that the strategy that `isTypeOf` uses basically just depends on the shape of the underlying data.

So in the error case `isTypeOf` will be handed an object that looks like:

```json
{
  "type": "error",
  "_type": "GravityMutationError",
  "message": "Artist Not Found",
}
```

So `(data) => data._type === "GravityMutationError"` serves as a simple test to identify this blob as an instance of a Gravity error.

Whereas in the success case it will be handed on object that looks like:

```json
{
  "id":  "abc123",
  "name": "Some Artist",
  "nationality": "Some Country",
  "birthday": 1942,
  "…": "…"
}
```

And the `(data) => data.id` test returns a truthy value, telling us that we were handed an instance of a database record, thus we presumably succeeded. 

Does that make sense?

Reply:
It does. Thanks for the explanation.

Reply:
thanks @ovasdi and @anandaroop . i learned!